src/util.cpp

```diff
@@ -893,15 +894,7 @@ void SetThreadPriority(int nPriority)
 #endif // WIN32
 }
 
-int GetNumCores()
-{
-#if BOOST_VERSION >= 105600
-    return boost::thread::physical_concurrency();
-#else // Must fall back to hardware_concurrency, which unfortunately counts virtual cores
-    return boost::thread::hardware_concurrency();
-#endif
-}
-
+int GetNumCores() { return std::thread::hardware_concurrency(); }
```

Comment:
I would like to see some sort of test to prove that our performance hasn't been been degraded..such as run a full sync with checkpoints turned off , with and without this change...and compare.